File: Hyeona/핸드폰_번호_가리기.java
package Hyeona;

public class 핸드폰_번호_가리기 {
    public String solution(String phone_number) {
        String answer = "";
        int length=phone_number.length()-4;
        for(int i=0;i<length;i++)
        {
            answer=answer+"*";
        }
        for(int i=length;i<length+4;i++)
        {
            answer=answer+phone_number.charAt(i);
        }
        return answer;
    }
}

File: Seonhong/핸드폰 번호 가리기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>


char* solution(const char* phone_number) {

    char* answer = (char*)malloc(1);
    strcpy(answer, phone_number);

    for (int i = 0; i < strlen(phone_number) - 4; i++) {

        answer[i] = '*';
    }


    return answer;

}